md_api/src/main/java/com/wipro/ats/bdre/md/api/TermJob.java

```diff
@@ -86,7 +84,8 @@ public TermJobInfo execute(String[] params) {
             //The TermJob completes even if sending message fails
             try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested try block into a separate method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1141)